libr/util/lib.c

```diff
@@ -303,19 +303,82 @@ R_API int r_lib_open_ptr (RLib *lib, const char *file, void *handler, RLibStruct
 
 	return ret;
 }
+#if __WINDOWS__ && !defined(__CYGWIN__)
+static char *utf16_to_utf8 (const wchar_t *wc) {
+	char *rutf8;
+	int csize;
+	
+	rutf8 = NULL;
+	csize = WideCharToMultiByte (CP_UTF8, 0, wc, -1, NULL, 0, NULL, NULL);
+	if (csize > 0) {
+		rutf8 = malloc (csize);
```

Comment:
doesn't malloc return NULL if it fails? I think it's enough to check if the return is NULL in the caller function 
btw, rutf8 = NULL in line 311 isn't necessary then right?